src/controllers/dynakube/version/version.go

```diff
@@ -80,19 +76,7 @@ func (r *Reconciler) updateImages(ctx context.Context, spec updateSpec) error {
 		}
 	}
 
-	if spec.updateEec {
-		err := imageUpdater.update(r.Dynakube.EecImage(), &r.Dynakube.Status.ExtensionController.VersionStatus, true)
-		if err != nil {
-			log.Error(err, "Failed to update Extension Controller image version")
-		}
-	}
-
-	if spec.statsdUpdate {
-		err := imageUpdater.update(r.Dynakube.StatsdImage(), &r.Dynakube.Status.Statsd.VersionStatus, true)
-		if err != nil {
-			log.Error(err, "Failed to update StatsD image version")
-		}
-	}
+	// TODO: should there be a synthetic update
```

Comment:
remove these synthetic todos, this will need to be done when there is full support for synthetic